eva-accession-release/src/main/resources/copy_release_to_ftp.sh

```diff
@@ -0,0 +1,226 @@
+UNMAPPED_VARIANTS_FOLDER=$4
+
+if [ ! -d ${OUTPUT_FOLDER} ]
+then
+  echo "Make sure that the output folder ${OUTPUT_FOLDER} exists. Aborting script, nothing was written."
+  exit 1
+fi
+
+mkdir ${INTERMEDIATE_FOLDER}
+if [ $? -eq 1 ]
+then
+  echo "Could not create intermediate folder ${INTERMEDIATE_FOLDER}. Aborting script, nothing was written."
+  exit 1
+fi
+
+# taken from vrnevadev:
+#
+# select distinct rs.assembly_accession, p.tax_id
+# from dbsnp_ensembl_species.rs_release_progress rs
+# join dbsnp_ensembl_species.import_progress p
+# on rs.dbsnp_db_name = p.database_name
+# where assembly_accession <> ''
+# order by assembly_accession, tax_id;
+echo "GCA_000001215.4	7227
+GCA_000001515.4	9598
+GCA_000001545.3	9601
+GCA_000001635.4	10090
+GCA_000001635.5	10090
+GCA_000001635.6	10090
+GCA_000001735.1	3702
+GCA_000001895.4	10116
+GCA_000002035.2	7955
+GCA_000002035.3	7955
+GCA_000002175.2	9598
+GCA_000002195.1	7460
+GCA_000002265.1	10116
+GCA_000002275.2	9258
+GCA_000002285.1	9615
+GCA_000002285.2	9615
+GCA_000002295.1	13616
+GCA_000002305.1	9796
+GCA_000002315.3	9031
+GCA_000002335.3	7070
+GCA_000002655.1	330879
+GCA_000002765.1	5833
+GCA_000002775.1	3694
+GCA_000002985.3	6239
+GCA_000003025.4	9823
+GCA_000003025.6	9823
+GCA_000003055.5	9913
+GCA_000003195.1	4558
+GCA_000003625.1	9986
+GCA_000003745.2	29760
+GCA_000004515.2	3847
+GCA_000004515.3	3847
+GCA_000004665.1	9483
+GCA_000005005.6	4577
+GCA_000005425.2	4530
+GCA_000005575.1	7165
+GCA_000146605.3	9103
+GCA_000146795.3	61853
+GCA_000148765.2	3750
+GCA_000151805.2	59729
+GCA_000151905.1	9593
+GCA_000181335.3	9685
+GCA_000188115.2	4081
+GCA_000188235.2	8128
+GCA_000219495.1	3880
+GCA_000224145.2	7719
+GCA_000233375.4	8030
+GCA_000237925.2	6183
+GCA_000247795.2	9915
+GCA_000247815.2	59894
+GCA_000298735.1	469796
+GCA_000298735.2	9940
+GCA_000309985.1	3711
+GCA_000317375.1	79684
+GCA_000317765.1	9923
+GCA_000317765.1	9925
+GCA_000331145.1	3827
+GCA_000355885.1	8839
+GCA_000364345.1	9541
+GCA_000372685.1	7994
+GCA_000409795.2	60711
+GCA_000413155.1	42345
+GCA_000442705.1	51953
+GCA_000686985.1	3708
+GCA_000695525.1	3712
+GCA_000710875.1	4072
+GCA_000772875.3	9544
+GCA_000966335.1	7950
+GCA_000987745.1	3635
+GCA_001433935.1	4530
+GCA_001465895.2	105023
+GCA_001522545.1	9157
+GCA_001577835.1	93934
+GCA_001625215.1	79200" > /tmp/assembly_to_taxonomy_map.txt
+
+echo -e "\nCopying to intermediate folder ${INTERMEDIATE_FOLDER}"
+# copy only relevant files. Any missing file will be listed in stderr. It's ok if some species don't have the *_issues.txt
+
+mkdir ${INTERMEDIATE_FOLDER}/${ALL_SPECIES_FOLDER}
+
+# copy unmapped_ids reports
+cd ${UNMAPPED_VARIANTS_FOLDER}
+tail -n +2 ${INPUT_FOLDER}/species_name_mapping.tsv |
+while read species_line
+do
+  taxonomy=`echo "${species_line}" | cut -f 5`
+  species_folder=`echo "${species_line}" | cut -f 1`
+  dbsnp_database_name=`echo "${species_line}" | cut -f 4`
+  mkdir ${INTERMEDIATE_FOLDER}/${ALL_SPECIES_FOLDER}/${species_folder}
```

Comment:
Use "mkdir -p" to ignore existing directory

Reply:
Same as in the other comment, intermediate folder should not exists so this should not cause any problem. In fact if this existed it could be an issue because we could be copying old data but with the validation at the beginning this should not happen